app/src/main/java/iam/thevoid/batteryviewexample/MainActivity.kt

```diff
@@ -19,11 +19,17 @@ class MainActivity : AppCompatActivity() {
         const val INITIAL_RED = 16
         const val INITIAL_BLUE = 16
         const val INITIAL_GREEN = 16
+        const val INITIAL_RED2 = 0
+        const val INITIAL_BLUE2 = 0
+        const val INITIAL_GREEN2 = 0
```

Comment:
MAY.
 
This is just an example app, but it would be nice if you'd replace all `...2` occurencies with `..._border_...`. For example `INITIAL_BORDER_GREEN`, 'blue_border_seek' etc.

Also would be great if you add TextView above each row of seekbars, that describes shortly what these seekbars for (Infill, Border)